README.md

```diff
@@ -289,6 +289,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Bhuvanesh Hingal](https://github.com/BhuvaneshHingal)
   - [Bhuvnesh Sharma](https://github.com/Bhuvnesh875)
   - [Bineet Naidu](https://github.com/bineetNaidu)
+  - [Bismeet Singh](https://github.com/BismeetSingh) 
```

Comment:
Please add your name in line 285, after Bishes Adhikari.

Reply:
Hey! @Panquesito7 looks like there is some order problem in the `B` list that's why I approved that.
```
  - [Biresh Biswas](https://github.com/Billa05)
  - [Bisesh Adhikari](https://github.com/Biseshadhikari)
  - [Benjamin Rukundo](https://github.com/rukundob451)
  - [Bhargav Avinash](https://github.com/bhargav794)
```
`I` after `e` and then `h`

Reply:
Made changes.

Reply:
You're right, looks like there's a problem here :eyes: What do you think we should do?

Reply:
For now, we can merge this request after that we can refactor the problem and push it :)